src/lib/leaflet-directive.js

```diff
@@ -4,7 +4,9 @@ import L from '@/lib/leaflet-utils/leaf'
 
 let map
 let layerGroup
-
+// TODO: replace  this with vue2-leaflet
```

Comment:
This leaflet-directive is a bad idea:
- not inspectable
- no hierarchy 
- not reusable

I believe the idea was to avoid a double administration of doms. Leaflet is responsible for building the dom elements in it's own component.  If you have a custom element (video, canvas, marker), then you can let vue take over.  You can always use Leaflet directly without a vue component by accessing the [mapObject](https://korigan.github.io/Vue2Leaflet/#/quickstart?id=accessing-leaflet-api)